index.d.ts

```diff
@@ -1,5 +1,14 @@
 declare namespace QuickLRU {
 	interface Options<KeyType, ValueType> {
+		/**
```

Comment:
This needs an actual description of the option first.

I also noticed we don't explain in what unit the max age.

Reply:
mm I don't follow you with the "I also noticed we don't explain in what unit the max age." what do you mean?